new-pipeline/src/dist.js

```diff
@@ -886,9 +886,12 @@ function displayHistograms(histogramsList, dates, useTable, cumulative, trim) {
     var description = histogramsList[0].histograms[0].description
     var metric = histogramsList[0].histograms[0].measure
     var channel = $("#channel-version").val().split("/")[0]
-    var desc = getDescriptionWithLink(metric, channel, description)
+    var desc = getDescription(metric, channel, description)
+    var link = getDescriptionLink(metric, channel, description)
 
     $('#dist-caption').html(desc);
+    $('#caption-dots').html("...");
+    $('#dist-caption-link').html(link);
```

Comment:
I think `text-overflow: ellipsis;` should already take care of adding the "...".

Reply:
@georgf , Okay, I will look into it and report ASAP. Thanks!